modules/e2ee/Context.js

```diff
@@ -260,9 +268,18 @@ export class Context {
                 return await this._decryptFrame(
                     encodedFrame,
                     keyIndex,
+                    initialKey || this._cryptoKeyRing[this._currentKeyIndex],
                     ratchetCount + 1);
             }
 
+            /*
+               Since the key it is first send and only afterwards actually used for encrypting, there were
```

Comment:
Can you please format this log with a `*` at the beginning of each line? LIke our other multiline logs please.